charts/eap-xp5/values.yaml

```diff
@@ -32,14 +32,14 @@ deploy:
     httpGet:
       path: /health/live
       port: admin
-    initialDelaySeconds: 60
+    initialDelaySeconds: 10
   readinessProbe:
     httpGet:
       path: /health/ready
       port: admin
     initialDelaySeconds: 10
   startupProbe:
     httpGet:
-      path: /health/live
+      path: /health/started
       port: admin
-    initialDelaySeconds: 60
\ No newline at end of file
+    initialDelaySeconds: 10
```

Comment:
With that setting, k8s will wait at least 10 seconds and at most 40 seconds (10 (initialDelaySeconds) + 3 (failureThreshold) * 10 (periodSeconds) to check that the server is started.
 
40 seconds might not be enough for some EAP XP applications and I'd prefer we extend that to 1 minute.
User have the ability to define their own thresholds but I'd prefer that by default, we wait more rather than not enough and risk false positive startup failure.

Could you add:

```
failureThreshold: 5
```

to your PR please?

Reply:
+1, done.